tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bookmark_menu.py

```diff
@@ -33,6 +33,10 @@ def tap_add_bookmark_to_home_screen_dialog_button(self):
         Wait(self.marionette).until(lambda m: self.apps.displayed_app.name != self.name)
         self.apps.switch_to_displayed_app()
 
+    def is_add_to_home_button_not_in_dialog(self):
```

Comment:
This function must return something. Used in `test_browser_bookmark.py`, it returns `None` which is considered to be [a false value](https://docs.python.org/2/library/stdtypes.html#truth-value-testing) by Python.

That's the reason why the test is passing. The problem is it can lead to a false positive. 

Here, I would use: 
``` Python
def is_add_to_home_button_in_dialog(self):
    return self.is_element_displayed(*self._add_bookmark_to_home_screen_dialog_button_locator)
```